tests/mirserver/URLBuilderDispatcher/urlbuilderdispatcher_test.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2020 UBports Foundation
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3, as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+ * SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "urlbuilderdispatcher.h"
+#include "mock_services.h"
+
+#include <QUrl>
+
+using namespace ::testing;
+
+class URLBuilderDispatcherTest : public ::testing::Test
+{
+protected:
+    void SetUp() override;
```

Comment:
Implement the function here to avoid needing to Implement it bellow

Do `void SetUp() override {}`